src/sys/mod.rs

```diff
@@ -12,7 +12,8 @@ pub mod eventfd;
 #[cfg(target_os = "linux")]
 pub mod memfd;
 
-#[cfg(not(any(target_os = "ios", target_os = "freebsd", target_os = "dragonfly")))]
+#[cfg(not(any(target_os = "freebsd", target_os = "dragonfly")))]
```

Comment:
It's fine to get rid of the "freebsd" here.  Probably fine to get rid of dragonfly, too, but I can't test on that platform.